Calculator_program.c

```diff
@@ -0,0 +1,127 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+
+#define MAX_LEN 100 // Maximum length
+
+// Function to calculate expressions 
+float calculateExpression(char *expr) {
+    float values[MAX_LEN]; 
+    char operators[MAX_LEN]; 
+    int valTop = -1; 
+    int opTop = -1; 
+
+    int i = 0;
+    //Function to perform operations
+    float applyOperation(float a, float b, char op) {

```

Comment:
Practice naming variables and function names such that they are self-descriptive and meaningful. This applies throughout the code, mentioning here only once.

Reply:
👍